compaction_picker.go

```diff
@@ -181,10 +181,8 @@ type pickedCompaction struct {
 	//    - in multilevel compaction, the output level is the lowest level involved in
 	//      the compaction
 	outputLevel *compactionLevel
-	// extraLevels contain additional levels in between the input and output
-	// levels that get compacted in multi level compactions
-	extraLevels []*compactionLevel
-	inputs      []compactionLevel
+	// Levels involved in the compaction in ascending order of depth.
```

Comment:
nit: `inputs contain levels involved in the compaction in ascending order of depth` (or something)